src/applications/static-pages/createAdditionalInfoWidget.js

```diff
@@ -41,7 +43,7 @@ export default function createAdditionalInfoWidget() {
         chevron.classList.toggle('open');
 
         if (analyticsEvent) {
-          recordEvent({ event: analyticsEvent });
+          recordEventOnce({ event: analyticsEvent });
```

Comment:
The best way to test this is probably to add a `console.log` right here and see that the event data is passed properly.

Reply:
Might be able to use this too: [Google Analytics debugger](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna)

Reply:
Oh cool, I didn't know about this. Added to Chrome 😄